### ISSUE TITLE
ego4d-nlq dataset

### ISSUE THREAD

**Issue by wenkangfan8-coder** [yeliudev/R2-Tuning#26]
Does the ego4d-nlq dataset not have a test set? The experimental results in the paper are not from the test set, are they?I noticed that in the annotation file you submitted, there are only "val" and "train". Is it possible to provide the test annotations?

**Comment by yeliudev** [comment]
The results in the paper are from the `val` set. Please refer to Ego4D's [official website](https://ego4d-data.org/) for the test set annotations.

**Comment by wenkangfan8-coder** [comment]
> The results in the paper are from the `val` set. Please refer to Ego4D's [official website](https://ego4d-data.org/) for the test set annotations.

First of all, I would like to express my sincere gratitude for your reply. I just checked the Ego4D dataset website. I noticed that the test set of the Ego4D-NLQ data subset seems to be "unlabeled". Only the train and val sets have been labeled, so it seems that we can't obtain the test results, right? I wonder if this is the reason why you didn't conduct the experiment on the test at that time?

**Comment by yeliudev** [comment]
Yes. We follow the previous works to test only on the validation set, as the results on test set requires online validation on Ego4D's server.
